docs/moleculenet.rst

```diff
@@ -119,6 +119,15 @@ Lipo Datasets
 
 .. autofunction:: deepchem.molnet.load_lipo
 
+Materials Datasets
+------------------
+Materials datasets include inorganic crystal structures, chemical
+compositions, and target properties like formation energies and band
+gaps.
+
```

Comment:
Could you add a paragraph expanding out information about the types of ML problems that commonly arise in materials science applications? It might be useful to compare/contract materials datasets with our other datasets and have a brief discussion about similarities and differences

Reply:
Will do!